VEHICLE/SetVehicleNameDebug.md

```diff
@@ -8,11 +8,8 @@ ns: VEHICLE
 void SET_VEHICLE_NAME_DEBUG(Vehicle vehicle, char* name);
 ```
 
-```
-NOTE: Debugging functions are not present in the retail version of the game.  
-```
+Assigns the name for the vehicle, viewable by selecting the "PedName" option within the debug widget's AI toggle display dropdown box. Note that debugging functions are not present in the retail version of the game.
```

Comment:
I think it's simply better to state that this is a debug method, only used in debug builds of the game. It may simply not be worth updating this native method's documentation at all (since it's already stated in the imported documentation in its fenced code-block).

Reply:
Would `This method is utilized solely for debugging purposes and is functional only in debug builds of the game. Please note that its functionality may not be available in the retail version.` be better then?

Reply:
> Would `This method is utilized solely for debugging purposes and is functional only in debug builds of the game. Please note that its functionality may not be available in the retail version.` be better then?

Yes.